apps/projects/app/components/Shared/FilterBar/FilterBar.js

```diff
@@ -402,116 +522,57 @@ const FilterBar = ({
-                <FilterByMilestone filters={filters} filtersData={filtersData} />
-                <OverflowDropDown>
-                  <FilterByStatus
-                    filters={filters}
-                    filtersData={filtersData}
-                    allFundedIssues={allFundedIssues}
-                    allIssues={allIssues}
-                  />
-                </OverflowDropDown>
-              </React.Fragment>
-            ) : (
-              <React.Fragment>
-                <FilterByProject filters={filters} filtersData={filtersData} />
-                <FilterByLabel filters={filters} filtersData={filtersData} />
-                <OverflowDropDown>
-                  <FilterByMilestone
-                    filters={filters}
-                    filtersData={filtersData}
-                  />
-                  <FilterByStatus
-                    filters={filters}
-                    filtersData={filtersData}
-                    allFundedIssues={allFundedIssues}
-                    allIssues={allIssues}
-                  />
-                </OverflowDropDown>
-              </React.Fragment>
-            )
-          )}
+          <Overflow filtersDisplayNumber={filtersDisplayNumber}>
+            <FilterByProject filters={filters} filtersData={filtersData} />
+            <FilterByLabel filters={filters} filtersData={filtersData} />
+            <FilterByMilestone filters={filters} filtersData={filtersData} />
+            <FilterByStatus
+              filters={filters}
+              filtersData={filtersData}
+              allFundedIssues={allFundedIssues}
+              allIssues={allIssues}
+            />
+          </Overflow>
         </FilterBarMainLeft>
 
-        <FilterBarMainRight>
-          {layoutName === 'large' ? (
-            <SearchInput
-              textFilter={textFilter}
-              updateTextFilter={updateTextFilter}
-            />
-          ) : (
-            <React.Fragment>
-              <Button icon={<IconSearch />} display="icon" onClick={activateSearch} ref={searchOpener} />
-              <SearchPopover
-                visible={searchVisible}
-                opener={searchOpener.current}
-                setSearchVisible={setSearchVisible}
-                textFilter={textFilter}
-                updateTextFilter={updateTextFilter}
-              />
-            </React.Fragment>
-          )}
+        <FilterBarMainRight ref={rightFBRef}>
+          <TextFilter
+            onClick={activateTextFilter}
+            textFilter={textFilter}
+            updateTextFilter={updateTextFilter}
+            visible={textFilterVisible}
+            openerRef={textFilterOpener}
+            setVisible={setTextFilterVisible}
+          />
 
-          <Button icon={<IconSort />} display="icon" onClick={activateSort} ref={sortersOpener} />
+          <Button icon={<IconSort />} display="icon" onClick={activateSort} ref={sortersOpener} label="Sorters" />
```

Comment:
"Sorters"? Is that the word used in the designs? "Sort by" sounds more expectable to me.

Reply:
Sure :)